agent/agente2e_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# TODO(lranjbar): Update to podman
```

Comment:
Will this followup in a subsequent PR? In dev-scripts podman it's more used

Reply:
Yeah so this is just my inexperience with podman showing. We should have a follow up task to convert it. 

I default to docker because it is what I'm used to. But I need to get more comfortable with podman in the future. Really I just used docker to get it done fast. Same reason why some people might just write a bash script really quickly.